client/js/components/day/day.js

```diff
@@ -12,32 +16,41 @@ var Day = React.createClass({
     // pending: array full of objects representing employees
     return {
       date: '',
-      gigs: [],
-      pending: []
+      gigs: {}
```

Comment:
the structure of the data changed and we now have an Employee store contains the employee data